armi/reactor/blocks.py

```diff
@@ -1462,9 +1467,11 @@ def getMfp(self, gamma=False):
         mfpNumerator = numpy.zeros(len(flux))
         absMfpNumerator = numpy.zeros(len(flux))
         transportNumerator = numpy.zeros(len(flux))
+
+        numDensities = self.getNumberDensities()
         # vol = self.getVolume()
         for nuc in self.getNuclides():
```

Comment:
can do similar single `getNuclides` and then use `getNuclideNumberDensities` as recommended above.